src/PolyhedralGeometry/markov_basis.jl

```diff
@@ -0,0 +1,68 @@
+@doc Markdown.doc"""
+    markov_basis(S; use_kernel=true)
+    markov_basis(P::Polyhedron{fmpq})
+
+
+Return a Markov basis for the lattice $L$. Where $L$ is
+- either the integer kernel of a matrix when either `use_kernel=true` or the
+  argument is of type `Polyhedron{fmpq}`,
+- otherwise $L$ is given by the span of the collection of integer vectors `S`.
+# Arguments
+- `S::AbstractVector{<:AbstractVector{U}} where U <: Union{Base.Integer, fmpz}`:
+  Collection of integer vectors that span the columnspace of a matrix M,
+  or when `use_kernel=false` collection of integer vectors that span an integer 
+  lattice.
+- `S::Union{MatElem{U}, <: AbstractMatrix{U}} where U <: Union{Base.Integer, fmpz}`:
+  A matrix whose integer kernel spans an integer lattice,
+  or when `use_kernel=false` a matrix whose rows span an integer lattice.
+- `P::Polyhedron{fmpq}`: Polyhedron whose homogenized lattice points span the columnspace of a matrix.
+
+
+# Algorithm
+Uses the project and lift algorithm see [HM05](@cite) and [DHK12](@cite)
+
+# Examples
+```jldoctest
+julia> S = [1 0 0 -1; 0 -1 1 0; -1 0 -1 0; 0 1 0 1]
+4×4 Matrix{Int64}:
+  1   0   0  -1
+  0  -1   1   0
+ -1   0  -1   0
+  0   1   0   1
+
+julia> M = markov_basis(S)
+Warning: unbounded / huge integer variable. Setting  <= (output unimplemented)
```

Comment:
That's maybe not ideal for an example in the reference manual :-)

Reply:
ya, this print is coming from polymake, I tried suppress the print, couldn't get it to work, we'll need to fix it for the next polymake release, I don't think we are in a rush to merge this before the renaming?